src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -355,7 +357,10 @@ var ReactDOM = {
     return renderSubtreeIntoContainer(null, element, container, callback);
   },
 
-  unstable_renderSubtreeIntoContainer(parentComponent : ReactComponent<any, any, any>, element : ReactElement<any>, containerNode : DOMContainerElement | Document, callback: ?Function) {
+  unstable_renderSubtreeIntoContainer(parentComponent : ReactComponent<any, any, any>,
+                                      element : ReactElement<any>,
+                                      containerNode : DOMContainerElement | Document,
+                                      callback: ?Function) {
```

Comment:
This isn't how we normally indent params. Check out [`createInstance `](https://github.com/facebook/react/blob/3c75593692524678fbb74e8077018cc39b04ffaa/src/renderers/dom/fiber/ReactDOMFiber.js#L171-L177) or [`prepareUpdate`](https://github.com/facebook/react/blob/3c75593692524678fbb74e8077018cc39b04ffaa/src/renderers/dom/fiber/ReactDOMFiber.js#L214-L221) above for examples.